readme.md

```diff
@@ -38,39 +39,40 @@ Hello, Node!
 
 * **Question:** Imagine you have a file called `classmates.txt`. For each part of CRUD, describe how the action would interact with the file.
 
-* **Your Answer:**
+* **Your Answer: C: create the text file, R: read the txt file, U: update (append) information to the txt file, and D: delete the txt file
 
 ---
 
 - [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
 
 * **Question:** What methods represent each CRUD action?
 
-* **Your Answer:**
+* **Your Answer:C:fs.writeFileSync() R:fs.readFileSync() U:fs.appendFileSync() D:fs.unlinkSync()
 
 ---
 
 - [ ] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
 
 * **Question:** What is the difference between these two methods?
 
-* **Your Answer:**
+* **Your Answer:one file is asynchronous and the readfilesync is not - it stops until it gets a call back.  
```

Comment:
This is not quite right. `fs.readFile()` is indeed asynchronous. Async functions do not necessarily wait _for_ the callback, they fire the callback once their work is done. `fs.readFileSync()` is synchronous and will block other processes from running until it finishes.